go_api/main.go

```diff
@@ -57,6 +58,14 @@ func main() {
 
 	// ルーティング(APIが増えると、server.goが肥大化するので、今後別にファイルに分ける)
 
+	// BasicAuth
+	e.Use(middleware.BasicAuth(func(username string, password string, c echo.Context) (bool, error) {
+		if username == "admin" && password == "password" {
+			return true, nil
+		}
```

Comment:
```suggestion
		if username == ADMIN && password == PASSWORD {
			return true, nil
		}
```
一応認証ということなので、環境変数に格納しますか。
この[記事](https://codelab.website/golang-godotenv/)が参考になります。
FinanSU, nutmeg-slackでも使っているのでわからない場合は聞いてください。